src/herder/test/TransactionQueueTests.cpp

```diff
@@ -17,6 +17,13 @@ using namespace stellar::txtest;
 
 namespace
 {
+enum class TransactionState
+{
+    UNKNOWN,
```

Comment:
same here, names are too short and have a high risk of conflicting with C style defines